Assets/Scripts/FpsController.cs

```diff
@@ -22,6 +22,9 @@ public class FpsController : MonoBehaviour
     public float forwardSpeed = 5f;
     [Range(0.1f, 15f)]
     public float backSpeed = 3f;
+    public Vector3 velocity { get { return _prevPosition - transform.position; } }
```

Comment:
nit: Suggest using shortand expression syntax for get-only properties

```suggestion
    public Vector3 velocity => _prevPosition - transform.position;
```